timesketch/lib/forms.py

```diff
@@ -0,0 +1,144 @@
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Form definitions and validators for the forms used in the application."""
+
+
+from flask_wtf import Form
+from wtforms import widgets
+from wtforms.fields import StringField
+from wtforms.fields import PasswordField
+from wtforms.fields import SelectMultipleField
+from wtforms.fields import RadioField
+from wtforms.fields import HiddenField
+from wtforms.fields import SelectField
+from wtforms.fields import BooleanField
+from wtforms.validators import DataRequired
+from wtforms.validators import Regexp
+from wtforms.validators import Length
+
+
+def build_form(request, form):
+    """WTForms need this in order to initialize the form."""
+    # TODO: Add link to why this is needed.
+    form_dict = MultiDict(request.json)
+    form_dict['csrf_token'] = request.headers.get('X-CSRFToken')
+    return form(form_dict)
+
+
+class MultiDict(dict):
+    """WTForms need this."""
```

Comment:
dito, describe what the class is used for in the docstring the fact that wtforms needs this I opt to use a comment.